python/dgl/graphbolt/impl/csc_sampling_graph.py

```diff
@@ -326,6 +326,48 @@ def _convert_to_sampled_subgraph(
             node_pairs=node_pairs, original_edge_ids=original_edge_ids
         )
 
+    def _convert_to_all_sampled_subgraph(
```

Comment:
What's the difference between this method with _convert_to_sampled_subgraph? This is a pretty poor name, can you come up with a better name, feel free to change _convert_to_sampled_subgraph if you need.